plutus-use-cases/test/Spec/GameStateMachine.hs

```diff
@@ -16,6 +16,7 @@ module Spec.GameStateMachine
   ( tests, successTrace, successTrace2, failTrace
   , prop_Game, propGame'
   , prop_NoLockedFunds
+  , prop_CheckNoLockedFundsProof
```

Comment:
are we running any of these tests?

Reply:
Not at the moment. The tests are a bit slow, so I was reluctant to add them to CI (but not running them is of course not good). What a reasonable time to spend running a property like this?